services/worker-manager/src/util.js

```diff
@@ -50,3 +50,21 @@ const createCredentials = (worker, expires, cfg) => {
   });
 };
 exports.createCredentials = createCredentials;
+
+const PAYLOAD_SENSITIVE_KEYS = ['staticSecret', 'secret'];
```

Comment:
we could probably add `accessToken` and `password` to that list, just to be safe, what do you think?

Reply:
I checked request payload schema in documentation and it doesn't list those, but we can include those, sure